server/package.json

```diff
@@ -24,7 +24,7 @@
   },
   "homepage": "https://github.com/MyLivingCity/my-living-city#readme",
   "dependencies": {
-    "@prisma/client": "^2.30.3",
+    "@prisma/client": "^4.13.0",
```

Comment:
Oh are we updating the Prisma and Nodemon versions in this PR?

Reply:
I think that was an automatic update when installing Prisma on my end. I'll check if it works with the previous version.